### PR TITLE
ffmpeg => 6.0

### PR DESCRIPTION
**Pull request by satmandu**
- This should not be merged before the following packages which claim to have dependencies on `ffmpeg` are updated/rebuilt/confirmed to work with `ffmpeg` 6.x:
- [x]  alsa_*
- [x]  chromaprint
- [x]  cmus ( added `opusfile` dep)
- [ ]  dosbox_x (does not build)
- [x]  ffmpegthumbnailer
- [x]  freerdp ( `openh264` and `cjson` deps added)
- [x]  gegl
- [x]  get_iplayer
- [x]  gifgen (shell script... doesn't need an update.)
- [x]  gstreamer
- [ ]  handbrake (does not build)
- [x]  libvncserver
- [x]  mpv
- [ ]  obs (needs libvlc package for update)
- [x]  peek
- [x] poppler (and `poppler_data`)
- [x]  qtfm
- [ ]  shotcut (updated but broken... need help fixing this.)
- [x]  spacefm _([This appears abandoned](https://github.com/IgnorantGuru/spacefm/issues/800) - last update was in 2018. I have used these patches: https://github.com/IgnorantGuru/spacefm/pull/749 https://github.com/IgnorantGuru/spacefm/pull/772 https://github.com/IgnorantGuru/spacefm/pull/781 https://github.com/IgnorantGuru/spacefm/pull/801 https://github.com/IgnorantGuru/spacefm/pull/813 )_
- [x]  waypipe
- [x] QT packages:
  - [x] qtbase
  - [x] qtdeclarative
  - [x] qtlocation
  - [x] qttools
  - [x] qtwebchannel
  - [x] qtmultimedia
  - [x] qtquickcontrols
  - [x] qtwayland
  - [x] qtx11extras
  - [x] qtwebengine (Currently a stub. Will be updated to the full version in a future PR)

(`nodebrew` is also fixed in this PR, as it is needed by the `qtwebengine` build...)

Builds properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=ffmpeg6 CREW_TESTING=1 crew update
```

### CODE REVIEW

**Comment by uberhacker** [comment]
It should be noted that updates to ffmpeg will also break ffmpegthumbnailer (at least it did last time) so that should be rebuilt too.

**Comment by satmandu** [comment]
Rebuilding ffmpeg 6 because man pages weren't installed. :/

**Comment by satmandu** [comment]
Failed builds
[dosbox_x-2023.05.01-x86_64.log](https://github.com/chromebrew/chromebrew/files/11492756/dosbox_x-2023.05.01-x86_64.log)
[obs-29.1.1-armv7l.log](https://github.com/chromebrew/chromebrew/files/11492757/obs-29.1.1-armv7l.log)

**Comment by satmandu** [comment]
Failed build...
[handbrake-1.6.1-x86_64.log](https://github.com/chromebrew/chromebrew/files/11492758/handbrake-1.6.1-x86_64.log)

**Review by uberhacker — APPROVED** [review]

File: packages/alsa_plugins.rb [hunk 1]
```diff
@@ -3,36 +3,37 @@
...
   description 'alsa-plugins contains plugins for various ALSA needs (e.g. Jack).'
   homepage 'https://www.alsa-project.org/main/index.php/Main_Page'
-  version '1.2.2'
+  version '1.2.7.1'
   license 'GPL-2 and LGPL-2.1'
-  compatibility 'all'
-  source_url 'ftp://ftp.alsa-project.org/pub/plugins/alsa-plugins-1.2.2.tar.bz2'
-  source_sha256 '1c0f06450c928d711719686c9dbece2d480184f36fab11b8f0534cb7b41e337d'
+  compatibility 'x86_64 aarch64 armv7l'
+  source_url 'http://www.alsa-project.org/files/pub/plugins/alsa-plugins-1.2.7.1.tar.bz2'
```
Comment (uberhacker) [inline packages/alsa_plugins.rb:9]: Change to use https?

File: packages/chromaprint.rb [hunk 2]
```diff
@@ -3,41 +3,39 @@
 class Chromaprint < Package
   description 'Chromaprint is a client-side library that implements a custom algorithm for extracting fingerprints from any audio source.'
   homepage 'https://acoustid.org/chromaprint'
-  version '1.5.1-aa67c95'
+  version '1.5.1-aa67c95-1'
   license 'LGPL-2.1'
-  compatibility 'all'
+  compatibility 'x86_64 aarch64 armv7l'
   source_url 'https://github.com/acoustid/chromaprint.git'
   git_hashtag 'aa67c95b9e486884a6d3ee8b0c91207d8c2b0551'
```
Comment (uberhacker) [inline packages/chromaprint.rb:10]: Newest git hash?

File: packages/alsa_plugins.rb (hunk 1 above)
Comment (satmandu) [reply packages/alsa_plugins.rb]: Done

File: packages/alsa_lib.rb [hunk 3]
```diff
@@ -3,35 +3,35 @@
 class Alsa_lib < Package
   description 'The Advanced Linux Sound Architecture (ALSA) provides audio and MIDI functionality to the Linux operating system.'
   homepage 'https://www.alsa-project.org/main/index.php/Main_Page'
-  version '1.2.4'
+  @_ver = '1.2.9'
+  version @_ver.to_s
```
[the remark this answers is not in the document]
Comment (satmandu) [reply packages/alsa_lib.rb]: Changed.

File: packages/ffmpeg.rb [hunk 4]
```diff
@@ -218,5 +220,8 @@ def self.build
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
     FileUtils.install 'tools/qt-faststart', "#{CREW_DEST_PREFIX}/bin/", mode: 0o755
+    FileUtils.mkdir_p "#{CREW_DEST_MAN_PREFIX}/man1/"
```
Comment (uberhacker) [inline packages/ffmpeg.rb:223]: Not needed with new FileUtils.install update?

File: packages/chromaprint.rb (hunk 2 above)
Comment (satmandu) [reply packages/chromaprint.rb:10]: Yes, to get it to build against ffmpeg 6...

File: packages/ffmpeg.rb (hunk 4 above)
Comment (satmandu) [reply packages/ffmpeg.rb]: Fixed.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**